lib/trento/infrastructure/commanded/middleware/enrich_register_application_instance.ex

```diff
@@ -36,7 +36,7 @@ defimpl Trento.Infrastructure.Commanded.Middleware.Enrichable,
          }}
 
       nil ->
-        {:error, :database_not_registered}
+        {:ignore, :database_not_registered}
```

Comment:
Let's add here, before returning, a concise message of what's happening, to give real undoubetable hint about what's their issue.
Something like:
```
Logger.error("database instance associated to application instance #{sid} not registered in Trento. Please make sure that Trento agent is running on database instances associated to this SAP system")
```
You can get the `sid` in the `RegisterApplicationInstance` command I guess

Reply:
Yes, maybe an warning is more appropriate